packages/react-notifications/src/components/notification/notification.tsx

```diff
@@ -25,6 +27,8 @@ import {
   getStylesForNotification,
 } from './notification.styles';
 
+const sequentialId = createSequentialId('notification-');
```

Comment:
FYI: for us to aware.

With React v18 there is a new hook `useId` that we should be able to use as a replacement for our custom implementation.

https://reactjs.org/blog/2022/03/29/react-v18.html#useid

Reply:
Our id generators was the first thing that came to my mind when I read about that new hook 😄